app/lib/services/handle_submission_for_store.rb

```diff
@@ -188,17 +188,15 @@ def funding_eligibility
       eligible_for_funding?
     end
 
```

Comment:
What happens if the transaction is rolled back and the job is still enqueued?

Reply:
In that situation then the Job wouldn't succeed, though with the Job being enqueued as the final step it's only a failure within the enqueuing that would prevent the transaction finalising.

Reply:
ok cool, was just checking 👍

Reply:
To be safer and to protect from future changes, we should move the enqueue outside the transaction 🙂 That way we don't accidentally add things after that in the block and cause issues.